composer.json

```diff
@@ -5,12 +5,12 @@
     "license": "MIT",
     "type": "symfony-bundle",
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8.0",
         "symfony/framework-bundle": "^4.4 || ^5.0",
         "ramsey/uuid": "^3.0 || ^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~7.5",
+        "phpunit/phpunit": "~7.5 || ^8.0 || ^9.0",
```

Comment:
This should just be upgraded to PHPUnit `^9.0` rather than allowing different versions. I'll need to look into that a bit more (probably).